docs/draft-release-notes.md

```diff
@@ -8,6 +8,7 @@
 - S3: Blocking public access to Amazon S3 storage. This feature provides settings for buckets to help manage public access to Amazon S3 resources. S3 Block Public Access settings override policies and permissions so that it is possible to limit public access to these resources.
 - S3: Server Side encryption can now be enabled and configured. This feature provides settings for configuring encryption of data in an S3 bucket.
 - S3: Object Lock. This feature allows storing objects using a write-once-read-many (WORM) model. Object Lock can help prevent objects from being deleted or overwritten for a fixed amount of time.
+- S3: Allow versioning updates. We add the ability to modify the versioning of a bucket the S3, to enable such functionality in step after its creation. Once versioning is enabled, it will no longer be disabled as the IaaS will throw an error.
```

Comment:
Slight typo - suggested amends:
`We add the ability to modify the versioning of an S3 bucket`
 `Once versioning is enabled, it can no longer be disabled`